### PR TITLE
Merge plus changelog

### PR DESCRIPTION
**Pull request by dirkhh**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x] New feature
- [ ] Code cleanup
- [ ] Build system change
- [x] Documentation change
- [ ] Language translation

### Pull request long description:

This brings in @djunkins ' dive site export change and a very short CHANGELOG for the monster merge

### Mentions:

@bstoeger this seems a very terse way to describe your 100 commits... feel free to suggest / add additional entries for CHANGELOG (you should be able to push / force push into this branch)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- Desktop: make dive sites 1st class citizens with their own dive site table
+- Desktop: only show dives at the dive sites selected in dive site table
+- Desktop: add undo functionality to edit operations and remove 'edited' state
```
Comment (bstoeger) [inline CHANGELOG.md:3]: That's an anticipating entry, as the edited state is still there for cylinder and weight-system editing.

Comment (dirkhh) [reply CHANGELOG.md:3]: true. I slightly overstated that. Feel free to edit / rewrite.

**Comment by tcanabrava** [comment]
I belive this series (while awesome, I must say. I was speechless when I
updated today) broke `Add Dive`, as it used to be, for historical reasons,
attached to the `editing` flag of the MainTab.

**Comment by dirkhh** [comment]
Ok, it seems for some reason @bstoeger is shy to edit my commit, not sure what to do with that.
I'll trash the CHANGELOG, have @bstoeger do his own CHANGELOG, and simply merge @djunkins ' commit into master.
At some point I'd love to understand why this ^^^ is...

**Comment by bstoeger** [comment]
Your CHANGELOG change is fine with me. After all, the edit-mode is removed from the notes-tab and undoizing the equipment tab is rather on the top of my todo-list. So far, I didn't find time to go through the monster commit. I hope you don't expect me to remember all the changes.

**Comment by dirkhh** [comment]
@tcanabrava add dive seems to still work for me. It's a bit awkward when you are used to the old system (because there is no way to abort - you have to undo your edits) and it's of course quite inconsistent (because the tank/weight thingy still has the abort option), but you can still manually add dives.
Can you say more what you think was broken (and we are of course discussing this on the completely wrong PR, but I wanted to comment where you had commented)

**Comment by dirkhh** [comment]
> I hope you don't expect me to remember all the changes.

@bstoeger I hate to point out the obvious... you MADE all these changes :-)
OK, I'll add back my partially incorrect CHANGELOG - it's better than nothing
(internet on the plane is surprisingly usable...)

**Comment by tcanabrava** [comment]
Dirk, I mean the editable points on the dive.

Em sáb, 13 de abr de 2019 às 17:33, Dirk Hohndel <notifications@github.com>
escreveu:

**Comment by dirkhh** [comment]
I just tested that - and it works as well. As @bstoeger said, we might want to move the accept / decline widget over to the profile for this, but in my (admittedly limited) testing, this does still seem to work.

**Pull request closed without merging**